efficient_r.Rmd

```diff
@@ -0,0 +1,476 @@
+# Effcient R
+
+Ruochen Zhang, Weipeng Li
+
+```{r setup, include=FALSE}
```

Comment:
Please use {r, include=FALSE} instead of {r setup, include=FALSE}

Thanks!

Reply:
I have changed the line into {r, include=FALSE}. Thank you!

Reply:
Thanks!